xbmc/linux/PlatformDefs.h

```diff
@@ -188,7 +188,6 @@ typedef unsigned short  WORD;
 typedef unsigned char   BYTE;
 typedef char        CHAR;
 typedef wchar_t     WCHAR;
-typedef int         BOOL;
```

Comment:
have you also completely rebuild depends?
https://github.com/xbmc/xbmc/blob/0f3ae09/tools/depends/native/TexturePacker/src/xwinapi.h#L28 uses `BOOL` from this file.